examples/scale/scenarios/scenarios.go

```diff
@@ -14,10 +14,50 @@
 
 package scenarios
 
-import "open-match.dev/open-match/pkg/pb"
+import (
+	"sync"
+
+	"github.com/sirupsen/logrus"
+	"google.golang.org/grpc"
+	"open-match.dev/open-match/examples/scale/scenarios/battleroyal"
+	"open-match.dev/open-match/examples/scale/scenarios/firstmatch"
+	"open-match.dev/open-match/internal/util/testing"
+	"open-match.dev/open-match/pkg/matchfunction"
+	"open-match.dev/open-match/pkg/pb"
+)
+
+var _ = firstmatch.Scenario()
+
+var (
+	queryServiceAddress = "om-query.open-match.svc.cluster.local:50503" // Address of the QueryService Endpoint.
+
+	logger = logrus.WithFields(logrus.Fields{
+		"app": "scale",
+	})
+)
 
 // ActiveScenario sets the scenario with preset parameters that we want to use for current Open Match benchmark run.
-var ActiveScenario = battleRoyalScenario
+var ActiveScenario = func() *Scenario {
+	var ls LogicalScenario = firstmatch.Scenario()
+
+	// TODO: Select which scenario to use based on some configuration or choice,
+	// so it's easier to run different scenarios without changing code.
+	ls = battleroyal.Scenario()
+
+	return &Scenario{
+		FrontendTotalTicketsToCreate: -1,
+		FrontendTicketCreatedQPS:     100,
+
+		BackendAssignsTickets: true,
+		BackendDeletesTickets: true,
+
+		Ticket:   ls.Ticket,
+		Profiles: ls.Profiles,
+
+		MMF:       queryPoolsWrapper(ls.MatchFunction),
+		Evaluator: ls.Evaluate,
+	}
+}()
 
 // Scenario defines the controllable fields for Open Match benchmark scenarios
 type Scenario struct {
```

Comment:
Now that I understand these, here are my 2 cents on naming :) Just reading without any pre-conceived notions.

I think BattleRoyale, FirstMatch ... fit well as a different packages that follow the Scenario interface. This structure (existing scenario) seems more appropriately as - Parameters / ScenarioParameters / ScenarioConfiguration / ScenarioOptions ... ?

Reply:
updated per discussion